CONTRIBUTING.md

```
@@ -0,0 +1,225 @@
+
+- [x] 🔵 **dynamic_analysis** — Dynamic analysis applied before major releases. *(SUGGESTED)*
+  - *Tool used:* Lighthouse & Chrome DevTools performance & accessibility scanner.
+
+- [x] 🔵 **dynamic_analysis_enable_assertions** — Dynamic analysis runs with assertions enabled. *(SUGGESTED)*
+  - *Note:* Strict React StrictMode enabled.
+
+- [x] 🔴 **dynamic_analysis_fixed** — Medium+ severity vulnerabilities fixed in a timely manner.
+  - *Note:* Verified clean.
+
+- [x] 🔵 **dynamic_analysis_unsafe** — Memory safety tools used. *(SUGGESTED)*
+  - *Note:* `[~]` N/A — Memory-safe TypeScript/JavaScript environment.
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)